docs/reference/sdk/interfaces/core.MultiHeaderTokenAuthentication.md

```diff
@@ -39,7 +39,7 @@ When requiresEndpointUrl is set to true this should be the root domain that all
 For example, this value would be "example.com" if specific endpoints looked like {custom-subdomain}.example.com.
 
 For packs that make requests to multiple domains (uncommon), this should be the domain within
-[networkDomains](core.PackVersionDefinition.md#networkdomains) that this configuration applies to.
+[networkDomains](core.PackDefinition.md#networkdomains) that this configuration applies to.
```

Comment:
Very interesting. I believe the `core.` prefix was needed then because the types crossed modules (`core` vs `development`), and I think it failed to resolve the link without it. With this PR they are resolving, just in strange ways.

Looking at the typedoc change log, I saw this entry:

> Added --useTsLinkResolution option (on by default) which tells TypeDoc to use TypeScript's @link resolution.

Does turning that off also resolve the issue? If not there are a number of other altered links in the `docs/` directory they also need addressing then.

Reply:
That *does* address the issue, though it comes with a couple other small changes, but I think they're fine.

Reply:
Ya, that LGTM. I'll take an action item to look into `--useTsLinkResolution ` and perhaps migrate to it in a future PR.